README.md

```diff
@@ -107,18 +107,21 @@ npm run euler
 ```
 ![image](https://github.com/theClubhouse-Augusta/project-euler/assets/124588/7a06281b-69f2-424d-8945-138259145f8f)
 
-### 5. Code Your Solution
-Start a new Project Euler solution by running the command `npm run euler` 
-or `node index.js`. This script will create all of the correct files and 
-folders, and will create a README file in your solution directory.
+### 5. Code Your Solution (for each problem)
+Start a new Project Euler solution by 1 of 2 options:
+
+   * run the command `npm run euler` 
+or `node index.js`
+
+   * provide command-line arguments to the script to skip 
+interactive mode<br>Examples:
```

Comment:
Why did you add an HTML line break here?

Reply:
To put the Examples on a new line. I can send you a screenshot.